cli/integration_tests/strict_env_vars/global_passthrough_env.t

```diff
@@ -0,0 +1,69 @@
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Repository Root: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Running command as global turbo (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::cli: pkg_inference_root set to "" (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: Found go binary at "[\-\w\/]+" (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: build tag: (go|rust) (re)
+  [-0-9:.TWZ+]+ \[INFO]  turbo: skipping turbod since we appear to be in a non-interactive context (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: global hash env vars: vars=\["VERCEL_ANALYTICS_ID"] (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: new global hash: value=ef18881222fd6a72 (re)
+  [-0-9:.TWZ+]+ |[DEBUG] turbo: local cache folder: path="" (re)
+  No tasks were executed as part of this run.
+
+Infer + []
+  $ ${TURBO} build --env=infer -vv --filter=nothing > /dev/null
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Global turbo version: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: No local turbo binary found at: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: No local turbo binary found at: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Repository Root: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Running command as global turbo (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::cli: pkg_inference_root set to "" (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: Found go binary at "[\-\w\/]+" (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: build tag: (go|rust) (re)
+  [-0-9:.TWZ+]+ \[INFO]  turbo: skipping turbod since we appear to be in a non-interactive context (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: global hash env vars: vars=\["VERCEL_ANALYTICS_ID"] (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: new global hash: value=9cda29cc8ed880a1 (re)
+  [-0-9:.TWZ+]+ |[DEBUG] turbo: local cache folder: path="" (re)
+  No tasks were executed as part of this run.
+
+Loose + []
+  $ ${TURBO} build --env=loose -vv --filter=nothing > /dev/null
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Global turbo version: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: No local turbo binary found at: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: No local turbo binary found at: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Repository Root: .* (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Running command as global turbo (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::cli: pkg_inference_root set to "" (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: Found go binary at "[\-\w\/]+" (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: build tag: (go|rust) (re)
+  [-0-9:.TWZ+]+ \[INFO]  turbo: skipping turbod since we appear to be in a non-interactive context (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: global hash env vars: vars=\["VERCEL_ANALYTICS_ID"] (re)
+  [-0-9:.TWZ+]+ \[DEBUG] turbo: new global hash: value=b7e2e8ed3383c982 (re)
+  [-0-9:.TWZ+]+ |[DEBUG] turbo: local cache folder: path="" (re)
+  No tasks were executed as part of this run.
+
+Loose + ["BONUS_VAR"]
```

Comment:
This has the same hash as the above one, since loose explicitly means "don't pay attention to it."